src/main/scala/com/amazon/deequ/analyzers/QuantileNonSample.scala

```diff
@@ -42,6 +42,23 @@ class QuantileNonSample[T](
 
   expand()
 
+  /** Given sketchSize, shrinkingFactor, and data, reconstruct the KLL Object */
+  def reconstruct(
+    sketchSizeInput: Int,
```

Comment:
you can just name these sketchSize etc and later use `this.sketchSize =  sketchSize` for disambiguation